### PR TITLE
Migrate from yarn to pnpm.

### PR DESCRIPTION
**Pull request by dessalines**

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 new_tag="$1"
 
-yarn version --new-version $new_tag
+pnpm version $new_tag
```
Comment (SleeplessOne1917) [inline deploy.sh:5]: Does pnpm have a version command? This was the one thing that tripped me up when I tried to migrate this to pnpm before.

File: package.json [hunk 2]
```diff
@@ -1,30 +1,20 @@
 {
   "name": "lemmy-js-client",
-  "version": "0.19.2-alpha.3",
   "description": "A javascript / typescript client for Lemmy",
-  "repository": "https://github.com/LemmyNet/lemmy-js-client",
-  "license": "AGPL-3.0",
+  "version": "0.19.2-alpha.3",
```
Comment (SleeplessOne1917) [inline package.json:4]: Out of scope with this PR, but how should we handle versioning for the JS client when it hasn't changed between the others repos version upgrade?

File: deploy.sh (hunk 1 above)
Comment (dessalines) [reply deploy.sh:5]: I'll test this now, but if it doesn't, npm version should work.

File: package.json (hunk 2 above)
Comment (dessalines) [reply package.json:4]: Really we should upgrade it either right before or after version release.

**Review by SleeplessOne1917 — APPROVED** [review]

**Pull request merged**